src/main/java/edu/iis/powp/adapter/IPlotterDrawPanelControllerAdapter.java

```diff
@@ -9,31 +9,36 @@
 /**
  * Plotter adapter to drawer with several bugs. 
  */
-public class MyAdapter extends DrawPanelController implements IPlotter
+public class IPlotterDrawPanelControllerAdapter extends DrawPanelController implements IPlotter
```

Comment:
Originally, 'I' prefix has been meant to suggest an interface. IPlotterDrawPanelControllerAdapter is a class so I would rather recommend PlotterDrawPanelControllerAdapter.